syft/pkg/cataloger/binary/cataloger_test.go

```diff
@@ -292,8 +292,8 @@ func Test_Cataloger_DefaultClassifiers_PositiveCases(t *testing.T) {
 			fixtureDir: "test-fixtures/classifiers/positive/python-binary-lib-3.7",
 			expected: pkg.Package{
 				Name:      "python",
-				Version:   "3.7.4a-vZ9",
-				PURL:      "pkg:generic/python@3.7.4a-vZ9",
+				Version:   "3.7.4",
```

Comment:
Was this something we wanted to no longer test as it's an invalid version number or was this part of the string causing the regular expression to fail to capture the version?

Reply:
As far as I know, this was an invalid version number, yes. The test was actually silently failing because it wasn't returning any packages.